### PR TITLE
September release 2021.09.100 (#7778)

### PR DESCRIPTION
**Pull request by DonJayamanne**

### CODE REVIEW

File: .github/release_plan.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 -   [ ] Update [Component Governance](https://dev.azure.com/vscode-python-datascience/vscode-python-datascience/_componentGovernance) (Click on "microsoft/vscode-jupyter" on that page). Notes are in the OneNote under Python VS Code -> Dev Process -> Component Governance.
     -   [ ] Provide details for any automatically detected npm dependencies
     -   [ ] Manually add any repository dependencies (if you can't add manually, refer [here](https://docs.opensource.microsoft.com/tools/cg/features/cgmanifest/)). Only add a cgmanifest.json if the components are not NPM or are not dev only.
--   [ ] Create new release branch with format `release-YYYY.MM.10`
+-   [ ] Create new release branch with format `release-YYYY.MM.100`
```
Comment (DonJayamanne) [inline .github/release_plan.md:11]: This was incorrect in the plan.

File: .github/workflows/build-test.yml [hunk 2]
```diff
@@ -77,11 +77,11 @@ jobs:
...
-      - name: insiders channel
-        # Scheduled builds will publish Insider builds.
-        if: github.event_name == 'schedule' && github.ref == 'refs/heads/main'
-        run: |
-          echo "release_channel=insider" >> $GITHUB_ENV
+    #   - name: insiders channel
+    #     # Scheduled builds will publish Insider builds.
+    #     if: github.event_name == 'schedule' && github.ref == 'refs/heads/main'
+    #     run: |
+    #       echo "release_channel=insider" >> $GITHUB_ENV
```
Comment (DonJayamanne) [inline .github/workflows/build-test.yml:84]: Disabled insiders

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,94 @@
...
+### Enhancements
+
+1. Split Notebook renderers into a separate extension ([Jupyter Notebook Renderers](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter-renderers)), allowing users to view Notebook outputs such as plotly, vega on [github.dev](https://github.dev).
+   ([#1909](https://github.com/Microsoft/vscode-jupyter/issues/1909))
+1. Added support to use Run by Line and Notebook Debugging in remote kernels.
+   ([#7576](https://github.com/Microsoft/vscode-jupyter/issues/7576))
+1. Added abiliyt to create notebooks using the menu options `File -> New File...`.
+   ([#7363](https://github.com/Microsoft/vscode-jupyter/issues/7363))
+1. Added a command to the command palette and an icon to the notebook toolbar to open the the table of contents for Notebooks.
+   ([#7305](https://github.com/microsoft/vscode-jupyter/issues/7305))
```
Comment (DonJayamanne) [inline CHANGELOG.md:14]: @rchiodo @IanMatthewHuff @DavidKutu Please review the news (this is now in the release branch)

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,94 @@
...
 
+## 2021.9.100 (6 October 2021)
+
+### Enhancements
+
+1. Split Notebook renderers into a separate extension ([Jupyter Notebook Renderers](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter-renderers)), allowing users to view Notebook outputs such as plotly, vega on [github.dev](https://github.dev).
+   ([#1909](https://github.com/Microsoft/vscode-jupyter/issues/1909))
+1. Added support to use Run by Line and Notebook Debugging in remote kernels.
+   ([#7576](https://github.com/Microsoft/vscode-jupyter/issues/7576))
+1. Added abiliyt to create notebooks using the menu options `File -> New File...`.
```
Comment (rchiodo) [inline CHANGELOG.md:11]: Spelling is off here.

**Review by rchiodo — APPROVED** [review]

**Pull request merged**